projects/cdk/utils/nil/is-nil.ts

```diff
@@ -0,0 +1,5 @@
+import {TuiNullable} from '@taiga-ui/cdk/interfaces';
+
+export function tuiIsNil<T>(value: TuiNullable<T>): value is null | undefined {
+    return value === null || typeof value === 'undefined';
```

Comment:
https://github.com/Tinkoff/taiga-ui/blob/11146b7b2dbdb613dd92c9f914df12996d446aaa/projects/cdk/utils/miscellaneous/is-present.ts#L5-L9

Reply:
Do we need both utils ?